azure-pipelines-integration.yml

```diff
@@ -43,24 +27,40 @@ jobs:
     - template: eng/pipelines/test-integration-job.yml
       parameters:
         configuration: Debug
-        oop64bit: true
+        oop64bit: false
         lspEditor: false
 
-- job: VS_Integration_Release_32
+- job: VS_Integration_Debug_64
   pool:
     name: $(poolName)
     demands: ImageOverride -equals $(queueName)
   timeoutInMinutes: 135
   variables:
     - name: XUNIT_LOGS
-      value: $(Build.SourcesDirectory)\artifacts\log\Release
+      value: $(Build.SourcesDirectory)\artifacts\log\Debug
   steps:
     - template: eng/pipelines/test-integration-job.yml
       parameters:
-        configuration: Release
-        oop64bit: false
+        configuration: Debug
+        oop64bit: true
         lspEditor: false
 
+- ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+  - job: VS_Integration_Release_32
```

Comment:
The title made me think we're doing debug _only_, but if we're still running at least one release that seems reasonable. I'd be worried to run _only_ debug since sometimes the release paths then get entirely untested.

Should this be release w/ OOP 64, since I'd imagine that's the most common customer configuration?

Reply:
Release builds still run on the main branch, just not on pull requests. I'd be happy to include the release 64 build (matches customers), but then we're at 3 builds per PR instead of 2.